pkg/oci/mutate/signatures.go

```diff
@@ -36,10 +39,21 @@ func AppendSignatures(base oci.Signatures, sigs ...oci.Signature) (oci.Signature
 			Annotations: ann,
 		})
 	}
+
 	img, err := mutate.Append(base, adds...)
+
+	imgCfg, _ := img.ConfigFile()
+
+	// Set the Created date to time of execution
+	if imgCfg.Created.Time.IsZero() {
+		log.Println("Signed Entry Date was Zero, Setting to current time")
```

Comment:
I don't think we need to log this, this can just be the new expectation.

Reply:
removed this if statement completely with your clarification that this should be covered in a unit test and always be updated 👍